.circleci/config.yml

```diff
@@ -83,12 +98,12 @@ jobs:
           command: |
             cd languages/java/secure-memory
             ./scripts/test.sh
+      - store_test_results:
```

Comment:
why did the order of this step change? the intent was to group test-related uploading logically at the end

Reply:
I changed it to group all the test related stuff and container file system stuff together.
And once everything is done, upload results is the last step.